3ds Max/Max2Babylon/Forms/ExporterForm.cs

```diff
@@ -196,7 +201,8 @@ private async Task<bool> DoExport(ExportItem exportItem, bool multiExport = fals
             exporter = new BabylonExporter();
             if (!string.IsNullOrWhiteSpace(txtTextureName.Text))
             {
-                exporter.relativeTextureFolder = Tools.GetPathRelativeToModel(Tools.UnformatPath(txtTextureName.Text), Tools.UnformatPath(txtModelName.Text));
+                // TODO:
```

Comment:
Oops, I need to re-add this for the next iteration.

Reply:
readded, this definitely exposed some additional spaghetti in our path related code. did some additional refactoring to remove duplicate functionality (we had two pretty similar functions for relative path formatting in Tools.cs) and moved the 3d software independent code to a Utilities class.